ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/metering_report_service.proto

```diff
@@ -50,6 +50,9 @@ message GetMeteringReportResponse {
 
     // The time at which the report was computed.
     google.protobuf.Timestamp report_generation_time = 3;
+
+    // The metering report json
```

Comment:
```suggestion
    // The metering report as JSON. Provided by all servers that support rendering the ``ParticipantMeteringReport`` to JSON.
```

Reply:
The protobuf ParticipantMeteringReport is not the same format as the output metering report json.  Do you think it is worth adding a Json Schema for the report format ?

Reply:
I think what's missing in particular is a clarification as to why the report is returned in two formats; and which report should be used by clients of the API. Adding a description or perhaps better a link to the description of the JSON format sounds very helpful as well. Could you make an attempt at adding such commentary?

Reply:
Agree. I've added https://digitalasset.atlassian.net/browse/DPP-1133 to cover this work.